modules/ha/pages/remove-failed-node.adoc

```diff
@@ -0,0 +1,69 @@
+= Removal of Failed Nodes
+:description: This page describes the procedure to remove a failed node.
+
+If a node fails in a highly available (HA) cluster (replication factor > 1), you can remove the failed node from the cluster while keeping all your data intact.
+After removal, use the xref:cluster-resizing:expand-a-cluster.adoc[cluster expansion] feature to restore your cluster to its original size.
+Restoring is important as node removal does not redistribute the data in your cluster.
+
+You should only consider removing a node when the node has failed.
+Removing a node from a cluster does not redistribute the data in the cluster.
+If all nodes are working in a cluster, and you want to reduce the size of the cluster, xref:cluster-resizing:shrink-a-cluster.adoc[shrink the cluster] instead.
+
+
+== Remove a single node
+If only a single node fails in a HA cluster, you can always remove the node without incurring any data loss.
+
+=== Prerequisites
+* Your cluster should have a replication factor greater than 1.
+* Service status in your TigerGraph cluster needs to meet the following requirements:
+You can check service status by running xref:system-management:management-with-gadmin.adoc#_gadmin_status[`gadmin status`].
++
+--
+** At least 1 `CTRL`(Controller) service is in `RUNNING` status
+** All `ZK`, `ETCD` and `EXE` services except those on the node being removed are in `RUNNING` status.
+--
+If the above services are down, and you cannot restart them, please contact mailto:support@tigergraph.com[TigerGraph Support].
+
+
+=== Procedure
+To remove a failed node from a cluster, run the following command and replace `<node_name>` with the name of the node and `node_ip_address` with the internal IP address of the node:
+
+[,console]
+----
+$ gadmin cluster remove <node_name>:<node_ip_address>
+----
+
+
+== Remove multiple nodes
+If there are multiple node failures in a cluster, recovering all data might not be possible.
+However, as long as there remains one complete replica of your data, it is possible to remove the failed nodes and restore the cluster afterwards with all data intact.
+
+Node removal is intended for machine failures.
+Concurrent failures of multiple nodes is an extremely unlikely event under normal circumstances.
+Therefore, the default Kafka configurations in TigerGraph do not prepare for multi-node failure.
+
+You can modify the default Kafka configurations to prepare for multi-node failure.
+However, you must do so before the failures happen.
```

Comment:
Yes, but we still have a tricky way to remove multi nodes even they did not prepare. They can ask for support if indeed happens.

Reply:
Added a note to ask user to contact support.